src/modules/dashboard/components/ColonyHome/ColonySubscription/ColonySubscription.tsx

```diff
@@ -100,24 +100,17 @@ const ColonySubscription = ({
             canUnsubscribe={isNetworkAllowed}
           >
             {({ isOpen, toggle, ref, id }) => (
-              <div
+              <ThreeDotsButton
                 id={id}
-                ref={ref}
+                innerRef={ref}
                 className={classnames(styles.menuIconContainer, {
                   [styles.menuActive]: isOpen,
                 })}
```

Comment:
You could also include the className in the `ThreeDotsButton`, using the `MemberActions` and `CommentsActions` styles as the default in `ThreeDotsButton.css`.

Then, for the `ColonySubscription` component, delete the shared styles from '.menuIconContainer' and pass it in as a `className` prop. Then add an `activeStyles` prop for `.menuActive`. Just remember to unset the border in the hover/active state and ensure the active fill is the correct colour.

This would allow you to delete shared styles from the respective CSS files and reduce code duplicate even further. 

ThreeDotsComponent:

```jsx
const { className, ...rest } = props;
  const active = activeStyles ? activeStyles : styles.menuActive;
  return (
    <Button
      className={classnames(styles.main, className, {
        [active]: isOpen,
      })}
      innerRef={innerRef}
      {...rest}
    >
      <Icon className={styles.icon} name='three-dots-icon' title={title} />
    </Button>
  );
```

ThreeDots styles (copied from MembersActions/CommentActions):
``` css
.main {
  padding: 0;
  height: 18px;
  width: 8px;
  position: relative;
  border: 1px solid transparent;
  border-radius: var(--radius-small);
  background: transparent;
  cursor: pointer;
}

.menuActive, .main:hover {
  border-color: var(--action-secondary);
}

.menuActive .icon, .main:hover .icon {
  fill: var(--action-secondary);
}

.main:focus {
  outline: 0;
}
```
ColonySubscription
``` jsx
<ThreeDotsButton
    id={id}
    innerRef={ref}
    isOpen={isOpen}
    className={styles.menuIconContainer}
    activeStyles={styles.menuActive}
    onClick={toggle}
    tabIndex={0}
    data-test="colonyMenuPopover"
    title={MSG.colonyMenuTitle}
  />
```

Reply:
@willm30 Great advice! I made the changes based on your suggestions and the button looks great. You can take a look at it again whenever you get a chance. Thank you!